<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValidateDialog</class>
 <widget class="QDialog" name="ValidateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log in...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="login_label">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>New Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="online_rb">
       <property name="text">
        <string>online</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="offline_rb">
       <property name="text">
        <string>offline</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="playername_label">
       <property name="text">
        <string>playername:</string>
       </property>
       <property name="buddy">
        <cstring>playername_le</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="playername_le"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>E-mail:</string>
       </property>
       <property name="buddy">
        <cstring>email_le</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>password:</string>
       </property>
       <property name="buddy">
        <cstring>password_le</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_le"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="showPassword_pb">
       <property name="text">
        <string>&amp;Show password</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="email_le"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="login_pb">
     <property name="text">
      <string>&amp;Log in</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>online_rb</tabstop>
  <tabstop>offline_rb</tabstop>
  <tabstop>playername_le</tabstop>
  <tabstop>email_le</tabstop>
  <tabstop>password_le</tabstop>
  <tabstop>login_pb</tabstop>
  <tabstop>showPassword_pb</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ValidateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ValidateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
